internal/apiclient/policyv2.go

```diff
@@ -41,6 +42,123 @@ func NewPolicyV2APIClient(ctx *config.RunContext) (*PolicyV2APIClient, error) {
 	return &c, nil
 }
 
+func (c *PolicyV2APIClient) CheckPolicies(ctx *config.RunContext, out output.Root) ([]output.TagPolicy, []output.FinOpsPolicy, error) {
```

Comment:
Can we change the signature to the following: 

```suggestion
type PolicyOutput struct {
    TagPolicies []output.TagPolicy
    FinnOpsPolicies []output.FinOpsPolicy
}

func (c *PolicyV2APIClient) CheckPolicies(ctx *config.RunContext, out output.Root) (*PolicyOutput, error) 
```

Probably personal preference, but, I prefer limit return args to maximium 2. This is because:

1. It's more verbose and readable for consumers to understand what's being returned from the function.
2. It's easier to add additional return args in the future by just appending to the output `struct`.
3. If we want to add additional logic to the return args (e.g. helper methods for filtering/comparison) it's easier to do with a struct, and easier for consumers to use.
4. If there's a complex/involved return arg we can document this more easily on as a parameter on the return struct.
5. we can return nil on an error case, which limits mem use (not super impactful here but could be if we add additional args in the future, see point 2)